tests/xapi-plugins/test_netdata.py

```diff
@@ -0,0 +1,26 @@
+class TestInstall:
+    def test_is_netdata_installed(self, host):
+        host.call_plugin('netdata.py', 'is_netdata_installed')
+
+    def test_install_netdata(self, host):
```

Comment:
Maybe we should assert that netdata is not installed already? Or make that a prerequisite through a fixture?

Reply:
And then we could also test what happens if you call install_netdata again when it's already installed. Two situations here by the way:
* netdata installed by install_netdata then try again
* netdata installed manually then try install_netdata (is the api key correctly set?)

Reply:
we always assume that no package are installed in other tests (mainly for storage drivers) so i think we should do the same here.
A fixture would be a good way to enforce that.

Reply:
> we always assume that no package are installed in other tests (mainly for storage drivers) so i think we should do the same here. A fixture would be a good way to enforce that.

In other tests, you don't have a plugin that may try to install things again, nor competition between users who main install it manually and the plugin that may install it, that's why I suggested that, after installing netdata once, we'd test what happens if you tell the plugin to install it again.

Reply:
I pushed a fixture enforcing netdata is NOT installed on the host. :+1: